src/avro_ocf.erl

```diff
@@ -146,12 +145,15 @@ make_header(Type) ->
 -spec make_header(avro_type(), extra_meta()) -> header().
 make_header(Type, ExtraMeta0) ->
   ExtraMeta = validate_extra_meta(ExtraMeta0),
```

Comment:
OK, that sounds reasonable. I'll prepare fix with your suggestions tomorrow morning.